spec/models/guide_import_spec.rb

```diff
@@ -115,7 +115,7 @@
       end
 
       describe 'it is removed from the guide' do
-        it { expect(guide.exercises.count).to eq 6 }
```

Comment:
This assert was wrong because initially the guide has 6 exercises (https://github.com/mumuki/mumuki-laboratory/blob/b879100110b943a17d82839d8740d065cde9722e/spec/models/guide_import_spec.rb#L75) and after importing and applying the json (https://github.com/mumuki/mumuki-laboratory/blob/b879100110b943a17d82839d8740d065cde9722e/spec/models/guide_import_spec.rb#L19) it should have 5 exercises